seed.js

```diff
@@ -0,0 +1,155 @@
+const mongoose = require('mongoose');
+const Promise = require('bluebird');
+const faker = require('faker');
+
+mongoose.connect('mongodb://localhost/description', {useUnifiedTopology: true, useNewUrlParser: true}).catch(err => console.log(err))
+
+var Schema = mongoose.Schema;
+
+var description = new Schema({
+  item_number: Number,
+  list_date: Date,
+  item_Spec: {
+    condition: String,
+    brand: String,
+    type: String,
+    packaging: String,
+    material: String,
+    rec_age: String,
+    char_family: String,
+    manfactured: String,
+    era: String,
+    year: Number,
+    size: Number,
+    upc: Number
+  },
+  seller_msg: {
+    prod_des: String,
+    item_des: String,
+    img_url: String
+  }
+})
+
+var ship_pay = new Schema ({
+  item_number: Number,
+  ship_handling: {
+    item_location: String,
+    ship_to: String,
+    ship_excludes: String,
+    qty: Number
+  },
+  shipping_cost: {
+    price: Number,
+    region:String,
+    service: String,
+    est_time: String
```

Comment:
I'd need to get more familiar with MongoDB, but does it have a data type for dates? Or is it really like a JS object so string is the appropriate data type?

Reply:
Yes there is, good catch, I updated the code to have Date as the expected value.